stats/stats.go

```diff
@@ -261,6 +261,21 @@ func (st *SampleTags) CloneTags() map[string]string {
 	return res
 }
 
+// ReplaceURLWithName replaces url tag with name tag, if both exist.
+// This method should on call for cloud output usage, see Issue #1166.
```

Comment:
Minor grammar issue here? "should on call"? Maybe "should be called" or "is called"?

Reply:
Agree, I should start reading the comments more thoroughly, both my own and in the PRs I review :blush:  I very much appreciate how often you find mistakes there! :heart:

Reply:
Fixed, I actually mean "should be called", not sure why "should on call" was written ^^

Reply:
@na-- Thanks, I'm a bit OCD about these details. :smile: It can get annoying in PRs, but good to know it's appreciated :)